torch_frame/nn/encoder/stype_encoder.py

```diff
@@ -182,6 +212,15 @@ def na_forward(self, feat: TensorData) -> TensorData:
                 fill_value = self.stats_list[col][StatType.MEAN]
             elif self.na_strategy == NAStrategy.ZEROS:
                 fill_value = 0
+            elif self.na_strategy == NAStrategy.NEWEST_TIMESTAMP:
+                fill_value = self.stats_list[col][StatType.NEWEST_TIME].to(
+                    feat.device)
+            elif self.na_strategy == NAStrategy.OLDEST_TIMESTAMP:
+                fill_value = self.stats_list[col][StatType.OLDEST_TIME].to(
+                    feat.device)
+            elif self.na_strategy == NAStrategy.MEDIAN_TIMESTAMP:
+                fill_value = self.stats_list[col][StatType.MEDIAN_TIME].to(
+                    feat.device)
```

Comment:
Can you check why it blames these two are not covered in the test? I think you've covered it, but there may be some error?

Reply:
Yeah this is interesting.

Reply:
Turns out that I missed some parens. Added~